CHANGELOG.md

```diff
@@ -12,6 +12,21 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## 7.5.5
+
+### Added
+
+* Added the ability to report a list of files and the reasons they couldn't be uploaded
+
+### Changed
+
+* Added better mapping of log levels to avoid aggressive logging from HTTPX and HTTPCORE
```

Comment:
Something like: "Improved logging when uplading files". No need to specify the libs here.